members/SOHL4908448c.yaml

```diff
@@ -0,0 +1,115 @@
+bioguide: SOHL4908448c
+contact_form:
+  method: post
+  action: ""
+  steps:
+    - visit: "http://www.ohiohouse.gov/c-allison-russo/contact"
+    - fill_in:
+        - name: tbxFirstName
+          selector: "#tbxFirstName"
+          value: $NAME_FIRST
+          required: true
+        - name: tbxLastName
+          selector: "#tbxLastName"
+          value: $NAME_LAST
+          required: true
+        - name: tbxEmail
+          selector: "#tbxEmail"
+          value: $EMAIL
+          required: true
+        - name: tbxAddress
+          selector: "#tbxAddress"
+          value: $ADDRESS_STREET
+          required: true
+        - name: tbxCity
+          selector: "#tbxCity"
+          value: $ADDRESS_CITY
+          required: true
+        - name: tbxZipCode
+          selector: "#tbxZipCode"
+          value: $ADDRESS_ZIP5
+          required: true
+        - name: tbxAreaCode
+          required: true
+          selector: "#tbxAreaCode"
+          value: "$PHONE_AREA_CODE"
+        - name: tbxPrefix
+          required: true
+          selector: "#tbxPrefix"
+          value: "$PHONE_EXCHANGE_CODE"
+        - name: tbxLineNumber
+          required: true
+          selector: "#tbxLineNumber"
+          value: "$PHONE_SUBSCRIBER_NUMBER"
+        - name: tbxSubject
+          selector: "#tbxSubject"
+          value: $SUBJECT
+          required: true
+        - name: tbxMessage
+          selector: "#tbxMessage"
+          value: $MESSAGE
+          required: true
+    - select:
+        - name: ddlState
+          selector: "#ddlState"
+          value: $ADDRESS_STATE_POSTAL_ABBREV
+          required: true
+          options:
+            AL: "AL"
+            AK: "AK"
+            AZ: "AZ"
+            AR: "AR"
+            CA: "CA"
+            CO: "CO"
+            CT: "CT"
+            DE: "DE"
+            DC: "DC"
+            FL: "FL"
+            GA: "GA"
+            HI: "HI"
+            ID: "ID"
+            IL: "IL"
+            IN: "IN"
+            IA: "IA"
+            KS: "KS"
+            KY: "KY"
+            LA: "LA"
+            ME: "ME"
+            MD: "MD"
+            MA: "MA"
+            MI: "MI"
+            MN: "MN"
+            MS: "MS"
+            MO: "MO"
+            MT: "MT"
+            NE: "NE"
+            NV: "NV"
+            NH: "NH"
+            NJ: "NJ"
+            NM: "NM"
+            NY: "NY"
+            NC: "NC"
+            ND: "ND"
+            OH: "OH"
+            OK: "OK"
+            OR: "OR"
+            PA: "PA"
+            RI: "RI"
+            SC: "SC"
+            SD: "SD"
+            TN: "TN"
+            TX: "TX"
+            UT: "UT"
+            VT: "VT"
+            VA: "VA"
+            WA: "WA"
+            WV: "WV"
+            WI: "WI"
+            WY: "WY"
+    - click_on:
```

Comment:
do we need any waits or finds?

Reply:
I don't think so. Tested these forms a few times to see if there was an issue related to waits or finds but didn't run into any. I could add a them tho, for extra precaution if we think it'd be valuable

Reply:
Nah, all good!